.github/workflows/build_libs.yml

```diff
@@ -160,7 +160,7 @@ jobs:
         cat <<EOF > env-vars
         TKLIB=${{ matrix.lib }}
         UPLOAD_PACKAGE=${UPLOAD_PACKAGE}
-        JFROG_ARTIFACTORY_TOKEN_1=${{ secrets.JFROG_ARTIFACTORY_TOKEN_1 }}
-        JFROG_ARTIFACTORY_USER_1=${{ secrets.JFROG_ARTIFACTORY_USER_1 }}
+        JFROG_ARTIFACTORY_TOKEN_2=${{ secrets.JFROG_ARTIFACTORY_TOKEN_2 }}
+        JFROG_ARTIFACTORY_USER_2=${{ secrets.JFROG_ARTIFACTORY_USER_2 }}
```

Comment:
Why are we not using the secrets directly?

Reply:
Because they are not directly accessible from within the `linuxbuildlib` script: they need to be passed as environment variables.